infra/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "infra",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "charts": "scripts/generate-chart-values.sh"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@types/jest": "^26.0.15",
+    "@types/js-yaml": "^3.12.5",
+    "@types/node": "^14.14.14",
+    "@zeit/ncc": "^0.22.3",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.4.4",
+    "ts-node": "^9.0.0",
+    "typescript": "4.0.5"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.813.0",
+    "js-yaml": "^4.0.0",
+    "yargs": "^16.2.0"
+  }
+}
```

Comment:
Should this be merged to the monorepo `package.json` ?
If yes then remove `yarn.lock`.

Reply:
No, this is completely separate set of dependencies which are only about the infra code generation and docker container.

Reply:
You could maybe pin the versions number of the dependencies